test/ui/__snapshots__/SelectRequest.test.tsx.snap

```diff
@@ -1,24 +1,6 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
+// Vitest Snapshot v1
 
-exports[`<SelectRequest /> allows to navigate the list of options using the arrow keys 1`] = `
-"❔   Select an option
-
- [34m❯[39m [34mItem 1[39m
-   Item 2
-   Item 3
-"
-`;
-
-exports[`<SelectRequest /> allows to navigate the list of options using the arrow keys 2`] = `
-"❔   Select an option
-
- [34m❯[39m [34mItem 1[39m
-   Item 2
-   Item 3
-"
-`;
-
-exports[`<SelectRequest /> displays the list of options filtering by the text input 1`] = `
+exports[`<SelectRequest /> > displays the list of options filtering by the text input 1`] = `
```

Comment:
seems to be obsolete snapshots from a skipped test. I was only able to update these after running `yarn test -u` opposed to `yarn test test/ui/SelectRequest.test.tsx -u`, maybe because of how vitest handles skipped tests?